dts/bindings/timer/riscv,machine-timer.yaml

```diff
@@ -1,13 +1,35 @@
 # Copyright (c) 2021 TOKITA Hiroshi <tokita.hiroshi@gmail.com>
 # SPDX-License-Identifier: Apache-2.0
 
-description: RISC-V Machine timer
+description: |
+  RISC-V Machine Timer
+
+  Privilege platforms provide a real-time counter, exposed as a 64-bit
+  memory-mapped machine-mode register named mtime. Another 64-bit register,
+  timer compare register (mtimecmp), causes a timer interrupt when the mtime
+  register contains a value greater than or equal to the value in the mtimecmp
+  register.
+
+  Example usage:
+
+    mtimer: timer@abcd1230 {
+      compatible = "riscv,machine-timer";
+      reg = <0xabcd1230 0x8,
+             0xabcd1238 0x8>;
+      reg-names = "mtime", "mtimecmp";
+    };
 
 compatible: "riscv,machine-timer"
 
 include: base.yaml
 
 properties:
+    reg:
+      required: true
+
+    reg-names:
```

Comment:
Add in `description` what the `reg-names` values are.

Reply:
There's an example in the description. Is it a good practice to override descriptions from fields that already do? This is a generic property already documented in `base.yaml`.